tests/laravel-test.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+LARAVEL_PROJECT="xammie/mailbook:dev-main"
```

Comment:
should we maybe specifically target a known good commit instead of a moving target?
fb2a633f49b47c02d3cee8ba1334f313791bd6b6

Reply:
Is this the good format?